openedx/adg/lms/applications/models.py

```diff
@@ -46,6 +50,20 @@ class UserApplication(TimeStampedModel):
     )
     cover_letter = models.TextField(blank=True, )
 
+    OPEN = 'Open'
+    ACCEPTED = 'Accepted'
+    WAITLIST = 'Waitlist'
```

Comment:
Can we please use lower cases for values? because these values might be using for some analytical purposes, it would be easier for them to write queries.